pkg/networkfs/resolvconf/resolvconf.go

```diff
@@ -12,9 +12,16 @@ import (
 )
 
 var (
-	defaultDns      = []string{"8.8.8.8", "8.8.4.4"}
-	localHostRegexp = regexp.MustCompile(`(?m)^nameserver 127[^\n]+\n*`)
-	nsRegexp        = regexp.MustCompile(`^\s*nameserver\s*(([0-9]+\.){3}([0-9]+))\s*$`)
+	// Note: the default IPv4 & IPv6 resolvers are set to Google's Public DNS
+	defaultIPv4Dns = []string{"8.8.8.8", "8.8.4.4"}
+	defaultIPv6Dns = []string{"2001:4860:4860::8888", "2001:4860:4860::8844"}
+	ipv4NumBlock   = `(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)`
+	ipv4Address    = `(` + ipv4NumBlock + `\.){3}` + ipv4NumBlock
+	ipv6Address    = `([0-9A-Fa-f]{0,4}:){2,7}([0-9A-Fa-f]{1,4})`
```

Comment:
@MalteJ @tianon given the complexity of an exact regex, I chose something that was a bit less aggressive given what we are coming from (at least from IPv4 nameserver perspective) was not even refusing invalid IPv4.. and given this particular code is not trying to explicitly validate, but rather match, it seemed close enough from all my investigation.  I did shorten it to not handle the IPv4-embedded, but that can be added back in to the regex if we feel that will be a valid use case.  Since we have dual-stack, not sure why embedded IPv4 would be that useful.